lib/posix/timer.c

```diff
@@ -29,7 +29,7 @@ struct timer_obj {
 };
 
 K_MEM_SLAB_DEFINE(posix_timer_slab, sizeof(struct timer_obj),
-		  CONFIG_MAX_TIMER_COUNT, 4);
+		  CONFIG_MAX_TIMER_COUNT, __alignof__(struct timer_obj));
```

Comment:
I think we use the `__alignof(type)` variant of this macro in Zephyr.

See 
https://github.com/zephyrproject-rtos/zephyr/blob/530a3092feef04bc9038f00ac088125b8514aba3/include/zephyr/toolchain/common.h#L195

Reply:
Nope - nevermind - we have both.